pkg/controller/instance/instance_controller.go

```diff
@@ -214,9 +202,9 @@ func (r *Reconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 		r.Recorder.Event(instance, "Normal", "PlanStarted", fmt.Sprintf("Execution of plan %s started", convert.StringValue(newExecutionPlan)))
 	}
 
-	// ---------- 4. If there's currently active plan, continue with the execution ----------
+	// ---------- 3. If there's currently active plan, continue with the execution ----------
 
-	activePlanStatus := getPlanInProgress(instance)
+	activePlanStatus := instance.GetScheduledPlan()
 	if activePlanStatus == nil { // we have no plan in progress
 		log.Printf("InstanceController: Nothing to do, no plan in progress for instance %s/%s", instance.Namespace, instance.Name)
```

Comment:
Nitpick: There's now a distinction between a scheduled plan and a plan in progress. Should that be reflected in the logs as well? I like that in the code "active" is used to described both states, we could do that here as well? The actual nitpick: This should read "no plan scheduled for instance" or "no plan active for instance".